scripts/prepublish.sh

```diff
@@ -2,6 +2,11 @@
 
 echo "pre-publish"
 
+if [ "$CI" = "true" ]; then
```

Comment:
No need to run this in CI since the tests will already have run in the Verify workflow